srml/babe/src/lib.rs

```diff
@@ -17,19 +17,21 @@
 //! Consensus extension module for BABE consensus.
 
 #![cfg_attr(not(feature = "std"), no_std)]
-#![forbid(unsafe_code, warnings)]
+#![forbid(unsafe_code)]
```

Comment:
Is this change intended? If so, why?

Reply:
Yes, to avoid breakage from future Rust versions.